ethcore/src/db.rs

```diff
@@ -212,7 +214,7 @@ impl Writable for DBTransaction {
 	}
 }
 
-impl Readable for Database {
```

Comment:
ideally we'd have the trait called `kvdb::Database`, but our rocksdb wrapper has already taken that name and I didn't want to break all existing usages of it (although the trait name `KeyValueDB` is arguably more descriptive). I could use a single-letter name for the generic param, but I don't think it really matters.